packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+/* Core root defaults */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// Remove all animations, transitions and smooth scroll
+// for people that prefer not to see them
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+/* Core body defaults */
+
+body {
+  background-color: var(--color-background);
+  color: var(--color-base);
+  font-family: var(--font-body);
+  font-size: var(--font-size-base);
+
+  // 👀 CMS-WDS/scout uses these reset styles
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  line-height: var(--font-line-height-base);
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+}
+
+// 👀 CMS-WDS/sep-screener uses this reset style
+// I've seen similar styles in other repos and I'm curious what it's solving
+#main[tabindex='-1']:focus,
+h1[tabindex='-1']:focus {
+  outline: 0;
+}
+*[tabindex='-1']:focus {
+  outline: 0;
+}
+
+/* Typography */
+// ❓ Do we want to enforce max-width rules on type els?
+// I personally feel like doing this would result in more support work for us
+// but it could result in better enforcement of this rule.
```

Comment:
That's a good question. @line47, @brentschneider, and @babsdenney, do you want to weigh in here?

Reply:
The current strategy in this reset is to automatically apply some standard styles to `h1` through `h6` and `p`, including standardized margin. The question posed here is whether we also want to put a width restriction like `measure--wide` on them as well, or if our users should always opt in to width restrictions.

Reply:
I'm inclined to vote _no_ since we don't currently have a nice way of opting out through a utility class. Maybe if we had a `ds-u-measure--none` that could remove it 🤔

Reply:
I'm also inclined to vote no on this unless we can talk to teams and find out if they want this change or not. Would teams be undoing it often?